src/V3AstNodes.cpp

```diff
@@ -1123,7 +1132,7 @@ void AstNodeCoverOrAssert::dump(std::ostream& str) const {
     if (immediate()) str << " [IMMEDIATE]";
 }
 void AstDisplay::dump(std::ostream& str) const {
-    this->AstNode::dump(str);
+    this->AstNodeStmt::dump(str);
```

Comment:
Probably would be good to put this into a little pull request to clean up.  If you would, please check all the other dumps for proper delegation too.